driver/main.c

```diff
@@ -498,26 +739,9 @@ static int ppm_open(struct inode *inode, struct file *filp)
 	reset_ring_buffer(ring);
 	ring->open = true;
 
-	if (g_tracepoints_attached == 0) {
```

Comment:
Behavioral break: if no tracepoint is attached, avoid enabling all of them. It is now up to the consumers to properly enable tracepoints.
Note: this is internally automatically managed by libscap kmod engine during start capture.